contributor/02-becoming-a-contributor.md

```diff
@@ -0,0 +1,45 @@
+
+#### Benefits to sharing solutions
+
+There is an African proverb stating that “If you want to go fast, go alone. If you want to go far, go together.” The same is true for software development teams:
+
+If you want to move fast, it's a great idea to break dependencies. The downside to that can be duplicated effort. In particular when re-implementing core logic there is the very real risk of the cost of duplication outweighing the benefit of independence.
+
+Collaborating with other teams enables you to share development cost. Just like in Open Source projects it can enable your team to build something bigger than you alone could have accomplished.
+
+But every team has a different roadmap? I've tried to use shared components before - they always took longer to integrate than it would have taken me to re-implement them. Those components were always lacking in some aspect or another - and my feature requests never got priority on the roadmap of the other team!
+
+In contrast to traditional project InnerSource projects come with the role of a Contributor: Yes, there will be times, when you wish that the shared solution has a new feature. As a Contributor you have the freedom to checkout the source code of the component, make modifications to it and use the resulting improved version.
+
+Yes, there will be times, when you urgently need a bug fix on your timeline that doesn't have the same priority in the host team. Becoming a Contributor enables you to become active yourself and support the host team with fixing that but.
+
+This way of working requires a change in mindset for many: Instead of waiting for features to be implemented or bugs fixed, instead of working around issues there's now a third solution. Spend your time and energy to check back with the InnerSource project on what your needs are - and then make the change directly in the shared project. So in addition to your coding skills, what you need to be successful in an InnerSource project are communication skills to clearly articulate your needs and come to a solution to address those that works for both, your team and the host team.
+
+"But I could simply go and fork the project, make my changes there and save myself from all this coordination overhead!" Sure - forking the project is a way to get your job done. Except in the long run this means it's on you to maintain that forked version for your team - and carry your changes forward for any new release the host team makes. Contributing your changes to the host team also means you get to benefit from their deeper knowledge of the component. They may spot issues in your patch, that otherwise would only have become obvious in production.
```

Comment:
```suggestion
"But I could simply go and fork the project, make my changes there and save myself from all this coordination overhead!". Sure - forking the project is a way to get your job done. Except in the long run this means it's on you to maintain that forked version for your team - and carry your changes forward for any new release the host team makes. Contributing your changes to the host team also means you get to benefit from their deeper knowledge of the component. They may spot issues in your patch, that otherwise would only have become obvious in production.
```

Reply:
Add full stop. `coordination overhead!" Sure` => `coordination overhead!". Sure`

Reply:
Thanks for having a look, Peter.